cfme/automate/dialogs/dialog_element.py

```diff
@@ -181,9 +182,17 @@ def create(self, element_data):
             view.fill(element)
             view.ele_save_button.click()
         view.save_button.click()
-        view.flash.assert_no_error()
-        view.flash.assert_message('{} was saved'.
-                                  format(self.parent.tab.dialog.label))
+        error_message = version.pick({
+            '5.8': 'Validation failed: Label is not unique within region 0',
+            '5.9': 'There was an error editing this dialog: '
+                   'Failed to create a new dialog - Validation failed: '
+                   'Name is not unique within region 0'})
+        if view.is_displayed:
```

Comment:
Isn't view the 'Add' destination view class here, and after a save it would be redirected to a different screen?  I'm thinking the flash message asserts should be in a `try|except` clause instead of dependent on the view.

If save was successful, the flash message assertions will not raise exception and the method can continue. If not, the flash message assert will raise the AssertionError anyway.